shell/platform/common/client_wrapper/include/flutter/event_stream_handler.h

```diff
@@ -20,8 +20,8 @@ struct StreamHandlerError {
   const std::string error_message;
   const std::unique_ptr<T> error_details;
 
-  StreamHandlerError(const std::string error_code,
-                     const std::string error_message,
+  StreamHandlerError(const std::string& error_code,
+                     const std::string& error_message,
```

Comment:
From my understanding, lines 26 and 27 will copy. Could we use `std::move` instead on lines 26 and 27 to allow the caller to avoid copies entirely?

Reply:
Same Loic, this is the result of clang_tidy --fix. A reference is not copied.

Reply:
I think changing the params here was a mistake when we fixed a serious bug with the ivars in https://github.com/flutter/engine/pull/40311.

While we could keep this signature to allow [copy elision](https://abseil.io/tips/117), it seems very unlikely that the performance of copying error strings (which are probably short, and shouldn't be frequent) is worth having non-standard argument types.

Reply:
Stuart, just to clarify, do you want this change reverted/changed to something else?

Reply:
I'm saying I think the removal of references on these two lines was probably a just mistake (cc @yaakovschectman in case I'm missing something looking back at that PR and review), and that the potential optimization Loic is suggesting here shouldn't be necessary here, and am thus voting for making the change you have here.

Reply:
Ack, thanks.